### DISCUSSION TITLE
VSCode icons not showing on search

### DISCUSSION THREAD

**Discussion by kaganergovan** [elrumo/macOS_Big_Sur_icons_replacements#1094] [Icon request] [answered]
Hi,
What happened to VSCode Icons? I always use VSCode icons. They don't appear on site. Will they be added again?

Regards,

**Comment by plamendobrev** [comment]
And my Mac though today is the day for the Visual Studio icon to disappear. Now my dock is ugly.

**Reply by kaganergovan** [reply]
Yess, same

**Comment by plamendobrev** [comment]
Something interesting: I managed to find some of the ones I liked manually, by scrolling through all of them. Not sure why they don't appear when searched.

**Reply by kaganergovan** [reply]
Same, it was really difficult to find out Docker in > 4k icons but finally I found it.

**Comment by elrumo** [comment]
That's really odd, I didn't remove them. I'll have a look what is happening, it seems to do something with the Algolia search

**Comment by elrumo** [comment, marked as answer]
Just fixed it, I'm not really sure what happened but one of the properties seemed to have been removed from some items on Algolia. All good now

**Reply by plamendobrev** [reply]
Thank you!

**Reply by kaganergovan** [reply]
Thanks!

**Reply by setanarut** [reply]
> Just fixed it, I'm not really sure what happened but one of the properties seemed to have been removed from some items on Algolia. All good now

VSCodium icons. They don't appear on site. Also searching for "vscode" should find "visual studio code" results.

<img width="367" alt="Ekran Resmi 2021-02-17 01 55 39" src="https://user-images.githubusercontent.com/36481442/108132215-8a877980-70c3-11eb-96d0-bd4429527d2f.png">
<img width="339" alt="Ekran Resmi 2021-02-17 01 55 52" src="https://user-images.githubusercontent.com/36481442/108132219-8bb8a680-70c3-11eb-8b52-bf2d1130c839.png">

**Comment by cake03** [comment]
Try searching for `VS code` instead of `vscode`

**Reply by elrumo** [reply]
I fixed it last morning, should be working fine again.
